enterprise/api/v1/serializers.py

```diff
@@ -952,3 +952,43 @@ def validate(self, data):  # pylint: disable=arguments-differ
         if not data.get('email') and not data.get('email_csv'):
             raise serializers.ValidationError('Must include either email or email_csv in request.')
         return data
+
+# pylint: disable=abstract-method
+class EnterpriseCustomerBulkSubscriptionEnrollmentsSerializer(serializers.Serializer):
+    """
+    Serializes an email field and for bulk enrollment requests.
+    """
+    emails = serializers.DictField(
+        child=serializers.DictField(
+            child=serializers.CharField(required=False, source='uuid'),
+        ),
+        required=True
+    )
+    courses = serializers.DictField(
+        child=serializers.ChoiceField(
+            choices=[
+                ("audit", _("Audit")),
+                ("verified", _("Verified")),
+                ("professional", _("Professional Education")),
+                ("no-id-professional", _("Professional Education (no ID)")),
+                ("credit", _("Credit")),
+                ("honor", _("Honor")),
+            ],
+            required=False,
+        ),
+        required=True,
+    )
+    reason = serializers.CharField(required=False)
```

Comment:
besides `courses` and `emails`, most of this is copied from the existing bulk enrollment endpoint.

Reply:
Why can you not _extend_ the existing class? Are there any items from the existing endpoint that do NOT exist on this new one?

Reply:
we don't use the fields 
```
    email = serializers.CharField(required=False)
    email_csv = Base64EmailCSVField(required=False)
    course_run_key = serializers.CharField(required=False)
    course_mode = serializers.ChoiceField(
```
None of those are required by default, but I feel it would be confusing to have seeming duplicates in a serializer class, no?